sources/assets/Xenko.Core.Assets/PackageSessionPublicHelper.cs

```diff
@@ -65,9 +65,11 @@ private static void CheckMSBuildToolset()
             {
                 if (projectCollection.Toolsets.Any(x => new Version(x.ToolsVersion).Major >= 15))
```

Comment:
If you don't mind, I would prefer the previous form to keep the flow simpler to understand, and easier to put additional checks following this one later.
You can and add a ! to negate the `Any`.